src/components/faq/FAQList.tsx

```diff
@@ -0,0 +1,184 @@
+import { useState } from 'react';
+import { Trash2 } from 'lucide-react';
+import { useCookies } from 'react-cookie';
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from '@/components/ui/accordion';
+import { useDeleteFaq } from '@/hooks/api/faq/useDeleteFaq';
+import useModal from '@/hooks/common/useModal';
+import { FAQListProps, FAQItemId } from '@/types/faq';
+import AlertDialog from '../common/AlertDialog';
+import Modal from '../common/Modal';
+
+export default function FAQList({
+  setIsEditing,
+  FAQ,
+  newFAQs,
+  setNewFAQs,
+  isEditing,
+}: FAQListProps) {
+  const safeFAQ: FAQItemId[] = Array.isArray(FAQ?.data) ? FAQ?.data : [];
+
+  const [cookies] = useCookies(['token', 'role']);
+  const { token } = cookies;
```

Comment:
다음과 같이 작성해도 좋을 것 같아요~!
```suggestion
  const [{ token }] = useCookies(['token', 'role']);
```